cmd/bosun/web/web.go

```diff
@@ -570,17 +570,18 @@ func APIRedirect(w http.ResponseWriter, req *http.Request) {
 
 var checkRunning = make(chan bool, 1)
 
-func Run(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interface{}, error) {
-	select {
-	case checkRunning <- true:
-		// Good, we've got the lock.
-	default:
-		return 0, fmt.Errorf("check already running")
-	}
-	d, err := schedule.Check(t, time.Now(), 0)
-	<-checkRunning
-	return d, err
-}
+//TODO: This makes less sense with every alert running independently.
```

Comment:
This can just be yanked as a feature. The reason I created it was so when someone "fixed" an alert, they didn't have to wait for it to return to normal. But because queries cover a time range, this is generally pretty futile. My idea to actually fix this problem is delayed close as described in https://github.com/bosun-monitor/bosun/issues/808